ethcore/src/client/client.rs

```diff
@@ -556,7 +564,7 @@ impl Client {
 			BlockID::Number(number) => Some(number),
 			BlockID::Hash(ref hash) => self.chain.block_number(hash),
 			BlockID::Earliest => Some(0),
-			BlockID::Latest => Some(self.chain.best_block_number())
+			BlockID::Latest | BlockID::Pending => Some(self.chain.best_block_number()),
```

Comment:
would be were it to exist. if we get to here we assume pending isn't available, in which case we return the latest.